### PR TITLE
Retired old endpoints

### PR DESCRIPTION
**Pull request by ywkim312**
This PR is deployed in dev instance

### CODE REVIEW

**Review by pengyin-shan — APPROVED** [review]
All API endpoints with `${config.apiUrl}/programs/` have been removed from the front-end code, as well as the 'map' endpoint of EQIP and CSP. Approving this PR.

File: app/pdl.yaml [hunk 1]
```diff
@@ -15,16 +15,28 @@ tags:
...
+  description: "Title II CSP conservation data."
+- name: "Title II: Conservation - CRP (Conservation Reserve Program)"
+  description: "Title II CRP conservation data."
+- name: "Title II: Conservation - ACEP (Agricultural Conservation Easement Program)"
+  description: "Title II ACEP conservation data."
+- name: "Title II: Conservation - RCPP (Regional Conservation Partnership Program)"
+  description: "Title II RCPP conservation data."
+- name: "Title XI: Crop insurance"
+  description: "Title XI Crop Insurance data."
+- name: "Title IV: SNAP (Supplemental Nutrition Assistant Program)"
```
Comment (sandeep-ps) [inline app/pdl.yaml:38]: It may be better to put Title IV name and description above Title XI for arranging the endpoints in the increasing order titles.

**Comment by sandeep-ps** [comment]
I have added couple of suggestions. The rest looks good to me. I briefly tested these endpoints using Postman. Thanks.

File: app/pdl.yaml (hunk 1 above)
Comment (ywkim312) [reply app/pdl.yaml:38]: You're right. Modified

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
```
[the remark this answers is not in the document]
Comment (ywkim312) [reply CHANGELOG.md:9]: changed to removed

**Review by sandeep-ps — APPROVED** [review]
Thanks for making the changes. Approving.

**Pull request merged**